docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -221,106 +221,139 @@ assumes that the Docker daemon is in the `stopped` state.
+		thin_pool_autoextend_percent=20
+	}
+	```
+
+11. Apply your new lvm profile
 
 	```bash
-	$ sudo lvcreate -L 4G -n metadata vg-docker
-	Logical volume `metadata` created.
+	$ lvchange --metadataprofile docker-thinpool docker/thinpool
 	```
 
-	This creates an LVM logical volume called `metadata` and an associated
-	block device file at `/dev/vg-docker/metadata`. In the next step you instruct
-	the `devicemapper` storage driver to use this block device to store image and
-	container metadata.
+12. Verify the `lv` is monitored.
+
+	```bash
+	$ lvs -o+seg_monitor
+	```
 
-7. Start the Docker daemon with the `devicemapper` storage driver and the
-`--storage-opt` flags.
+13. If the Docker daemon was previously started, clear your graph driver directory.
 
-	The `data` and `metadata` devices that you pass to the `--storage-opt`
-	options were created in the previous steps.
+	Clearing your graph driver removes any images, containers, and volumes in your
+	Docker installation.
 
 	```bash
-	$ sudo docker daemon --storage-driver=devicemapper --storage-opt dm.datadev=/dev/vg-docker/data --storage-opt dm.metadatadev=/dev/vg-docker/metadata &
-	[1] 2163
-	[root@ip-10-0-0-75 centos]# INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
-	INFO[0027] Option DefaultDriver: bridge
-	INFO[0027] Option DefaultNetwork: bridge
-	<-- output truncated -->
-	INFO[0027] Daemon has completed initialization
-	INFO[0027] Docker daemon commit=1b09a95 graphdriver=aufs version=1.11.0-dev
+	$ rm -rf /var/lib/docker/*
 	```
 
-	It is also possible to set the `--storage-driver` and `--storage-opt` flags
-	in the Docker config file and start the daemon normally using the `service` or
-	`systemd` commands.
+14. Configure the Docker daemon with specific devicemapper options.
 
-8. Use the `docker info` command to verify that the daemon is using `data` and
-`metadata` devices you created.
+	There are two ways to do this. You can set options on the commmand line if you start the daemon there:
 
 	```bash
-	$ sudo docker info
-	INFO[0180] GET /v1.20/info
-	Containers: 0
-	Images: 0
-	Storage Driver: devicemapper
-	 Pool Name: docker-202:1-1032-pool
-	 Pool Blocksize: 65.54 kB
-	 Backing Filesystem: xfs
-	 Data file: /dev/vg-docker/data
-	 Metadata file: /dev/vg-docker/metadata
-	[...]
+	--storage-driver=devicemapper --storage-opt=dm.thinpooldev=/dev/mapper/docker-thinpool --storage-opt dm.use_deferred_removal=true
 	```
 
-	The output of the command above shows the storage driver as `devicemapper`.
-	The last two lines also confirm that the correct devices are being used for
-	the `Data file` and the `Metadata file`.
+	You can also set them for startup in the `daemon.json` configuration, for example:
+
+	```json
+	 {
+		 "storage-driver": "devicemapper",
+		 "storage-opts": [
+			 "dm.thinpooldev=/dev/mapper/docker-thinpool",
+			 "dm.use_deferred_removal=true"
+		 ]
+	 }
+	```
+15. Start the Docker daemon.
+
+	```bash
+	$ systemctl start docker
```

Comment:
Perhaps we should add a `systemctl daemon-reload` here to be sure changes are loaded (if they used a drop-in file, or modified the docker.service)